### PR TITLE
Generate Data Package DAGs

### PR DESCRIPTION
**Pull request by cpcundill**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Added generation of data package DAGs.  Moved some re-usable stuff to utils

## Related Tickets & Documents

- Ticket Link: https://github.com/orgs/digital-land/projects/9/views/3?pane=issue&itemId=85558498&issue=digital-land%7Ctechnical-documentation%7C143

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [x] No, and this is why: will run manually via GHA and verify within Airflow UI
- [ ] I need help with writing tests

## GHA

Manually triggered: https://github.com/digital-land/airflow-dags/actions/runs/11668125801

## Screenshots

Trigger DAG now includes organisation-builder, which becomes the new pre-requisite for the rest of the DAGs:

Screenshot 2024-11-04 at 16 14 19

## [optional] Are there any dependencies on other PRs or Work?

Yes: https://github.com/digital-land/digital-land-infrastructure/pull/156

### CODE REVIEW

**Comment by cpcundill** [comment]
Will publish manually via GHA and verify within Airflow UI

File: dags/data_package_generator.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    with DAG(
+            f"{package}-builder",
+            default_args=dag_default_args,
+            description=f"Data package builder task for the {package} data package",
+            schedule=None,
+            catchup=False,
+            params={
+                "cpu": Param(default=8192, type="integer"),
+                "memory": Param(default=32768, type="integer"),
+                "transformed-jobs": Param(default=8, type="integer"),
```
Comment (eveleighoj) [inline dags/data_package_generator.py:32]: These are specific to the collection task so while they can be fed in without causing problems they aren'r rally doing much here. They essentially tell make how many parallel jobs to run at once.

Comment (cpcundill) [reply dags/data_package_generator.py]: Cheers - removed

**Comment by cpcundill** [comment]
Example of successful run: https://<dev airflow>/dags/organisation-builder/grid?dag_run_id=manual__2024-11-04T19%3A29%3A11%2B00%3A00&task_id=build-data-package&tab=logs

**Review by cjohns-scottlogic — APPROVED** [review]

**Pull request merged**